fov_processing_pipeline/data/quilt.py

```diff
@@ -0,0 +1,67 @@
+import os
+
+import quilt3
+
+from . import utils as data_utils
+
+
+def get_data(save_dir=None, n_fovs=100, protein_list=None, overwrite=False, use_current_results=False):
+    """
+    Function to pull data from quilt3 and return pandas dataframes containing per-fov and per-cell info
+
+    Parameters
+    ----------
+    save_dir: str
+        save directory of images and metadata
+
+    n_fovs: int
+        Number of images per protein to retain
+
+    protein_list:
+        Protein names to retain
+
+    overwrite: bool
+        do we overwrite the files if they exist? (i.e. do you want to put new results in an old directory)
+
+    use_current_results: bool
+        do we skip downloading data, and just use whatever data already present on disc
+
+    Returns
+    -------
+    cell_data: pandas.DataFrame
+        Dataframe where each row corresponds to a single cell
+
+    fov_data: pandas.DataFrame
+        Dataframe where each row corresponds to an FOV
+    """
+
+    aics_pipeline = quilt3.Package.browse(
+        "aics/pipeline_integrated_cell", registry="s3://allencell"
+    )
+
+    metadata_fn = aics_pipeline["metadata.csv"]
+
+    cell_data = metadata_fn() # noqa
```

Comment:
This is a 'quilt-ism', that not many people know, maybe add a comment here explaining that quilt has built in deserializers for some data formats